sdk/inc/azure/storage/az_storage_blobs.h

```diff
@@ -33,16 +33,24 @@
  */
 static az_span const AZ_STORAGE_API_VERSION = AZ_SPAN_LITERAL_FROM_STR("2019-02-02");
 
+/**
+ * @brief Azure Storage Blobs blob client options
```

Comment:
nit:
```suggestion
 * @brief Azure Storage Blobs #az_storage_blobs_blob_client options
```

Reply:
I think this comment is redundant/obvious given the name. Merge it with the remark.

```suggestion
/**
 * @brief Allows the user to define custom behavior of the #az_storage_blobs_blob_client.
 *
 */
```

Or
```suggestion
/**
 * @brief Allows customization of the #az_storage_blobs_blob_client.
 *
 */
```